### PR TITLE
add documentation for flex methods

### PR DESCRIPTION
**Pull request by XcrossD**
Follow up from:
https://github.com/Yoctol/messaging-apis/pull/381

### CODE REVIEW

File: content/docs/APIReference-LineContext.md [hunk 1]
```diff
@@ -480,6 +482,66 @@ context.replyImageCarouselTemplate('this is an image carousel template', [
 
 <br />
 
+### Reply Flex Messages
+
+## `replyFlex(altText, contents)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#flex-message)
```
Comment (kpman) [inline content/docs/APIReference-LineContext.md]: @XcrossD Thanks for sending this PR.
We use H4 markdown format (`####`) for method API.
Could you modify them to fit the convention?

File: content/docs/APIReference-LineContext.md [hunk 2]
```diff
@@ -480,6 +482,66 @@ context.replyImageCarouselTemplate('this is an image carousel template', [
...
+
+| Param    | Type     | Description                                                                                             |
+| -------- | -------- | ------------------------------------------------------------------------------------------------------- |
+| altText  | `String` | Alternative text.                                                                                       |
+| contents | `Object` | Flex Message [container](https://developers.line.me/en/docs/messaging-api/reference/#container) object. |
+
+Example:
+
+```js
+client.replyFlex('this is a flex', {
```
Comment (tw0517tw) [inline content/docs/APIReference-LineContext.md]: Please change this to `context.replyFlex`.

File: content/docs/APIReference-LineContext.md [hunk 3]
```diff
@@ -951,6 +1013,66 @@ context.pushImageCarouselTemplate('this is an image carousel template', [
...
+
+| Param    | Type     | Description                                                                                             |
+| -------- | -------- | ------------------------------------------------------------------------------------------------------- |
+| altText  | `String` | Alternative text.                                                                                       |
+| contents | `Object` | Flex Message [container](https://developers.line.me/en/docs/messaging-api/reference/#container) object. |
+
+Example:
+
+```js
+client.pushFlex('this is a flex', {
```
Comment (tw0517tw) [inline content/docs/APIReference-LineContext.md]: Please change this to `context.pushFlex`.

**Comment by tw0517tw** [comment]
Great! Thank you!

**Pull request merged**